site/_docs/reference.md

```diff
@@ -2649,7 +2649,7 @@ BigQuery's type system uses confusingly different names for types and functions:
 * Similarly, `DATETIME(string)` returns a Calcite `TIMESTAMP`.
 
 | C | Operator syntax                                | Description
-|:- |:-----------------------------------------------|:-----------
+|: |:-----------------------------------------------|:-----------
```

Comment:
what happens when you remove this minus?

Reply:
Resolved.